src/WebDocs/wwwroot/app/functions/GetWindow/samples/001/content.html

```diff
@@ -0,0 +1,3 @@
+<div d-dataKey="windowData" d-dataType="object" d-dataProperty-currentWindow=""></div>
+<input type="button" value="Get Current Window" d-on-click="UpdateDataField(windowData,[currentWindow],GetWindow())" />
```

Comment:
The function GetWindow will only work if you are inside a window. You should open a window and inside the window cal the function